test/units/res.set.test.ts

```diff
@@ -0,0 +1,122 @@
+import opine from "../../mod.ts";
+import { describe, it } from "../utils.ts";
+import { superdeno } from "../deps.ts";
+
+describe("res", function () {
+  describe(".set(field, value)", function () {
+    it("should set the response header field", function (done) {
+      var app = opine();
+
+      app.use(function (req, res) {
+        res.set("Content-Type", "text/x-foo; charset=utf-8").end();
+      });
+
+      superdeno(app as any)
```

Comment:
I _think_... I patched the type issues with `superdeno` with an upgrade - is it still throwing a type error?